pkg/apis/extensions/v1beta1/conversion.go

```diff
@@ -41,6 +41,12 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 		Convert_v1beta1_DeploymentStrategy_To_extensions_DeploymentStrategy,
 		Convert_extensions_RollingUpdateDeployment_To_v1beta1_RollingUpdateDeployment,
 		Convert_v1beta1_RollingUpdateDeployment_To_extensions_RollingUpdateDeployment,
+		Convert_extensions_DaemonSetSpec_To_v1beta1_DaemonSetSpec,
```

Comment:
Hmm, this is part from rebased  #19627. I didn't check if it still needed. I will check it.